packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -53,10 +53,40 @@ new lambda.NodejsFunction(this, 'my-handler', {
 ```
 
 ### Configuring Parcel
-The `NodejsFunction` construct exposes some [Parcel](https://parceljs.org/) options via properties: `minify`, `sourceMaps`,
-`buildDir` and `cacheDir`.
+The `NodejsFunction` construct exposes some [Parcel](https://parceljs.org/) options via properties: `minify`, `sourceMaps` and `cacheDir`.
 
 Parcel transpiles your code (every internal module) with [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) and uses the
 runtime version of your Lambda function as target.
 
 Configuring Babel with Parcel is possible via a `.babelrc` or a `babel` config in `package.json`.
+
+### Working with modules
+
+#### Externals
+By default, all node modules are bundled except for `aws-sdk`. This can be configured by specifying
+the `externalModules` prop.
+
+```ts
+new lambda.NodejsFunction(this, 'my-handler', {
+  externalModules: [
+    'aws-sdk', // Use the 'aws-sdk' available in the Lambda runtime
+    'cool-module', // 'cool-module' is already available in a Layer
+  ],
+});
+```
+
+#### Install modules
+Use the `installModules` prop to specify a list of modules that should not be bundled
```

Comment:
Add a preface saying: "By default, Parcel will automatically detect all modules referenced in your lambda code and will include them in the lambda bundle. ..."